packages/pages/src/common/src/template/internal/types.ts

```diff
@@ -133,23 +135,35 @@ export const parse = (
   };
 };
 
+export const isStaticTemplateConfig = (
```

Comment:
Could you import this from packages/pages/src/common/src/feature/features.ts instead of making a copy?